chore(docs): run-node update informations (#35)
atomone-hub/atom.one: content/english/run-node.md

[https://atomone.fra1.digitaloceanspaces.com/atomone-1/genesis.json](https://atomone.fra1.digitaloceanspaces.com/atomone-1/genesis.json)

> ## Networks informations
> 
> All the networks informations, versions and upgrade are gather in [github.com/atomone-hub/networks](https://github.com/atomone-hub/networks)

Reviewer: ```suggestion
## Network information

All network information, versions and upgrades are available at [github.com/atomone-hub/networks](https://github.com/atomone-hub/networks)
```

### Setting recommendations

> You can find community informations on [cosmos.directory/atomone](https://cosmos.directory/atomone).

Reviewer: Community settings can be found at [cosmos.directory/atomone](https://cosmos.directory/atomone).

> You can find community informations on [cosmos.directory/atomone](https://cosmos.directory/atomone).

Reviewer: ```suggestion
Community settings can be found at [cosmos.directory/atomone](https://cosmos.directory/atomone).
```

| Setting            | Recommendation                                                                                                                  |
| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
| minimum-gas-prices | 0.025uatone                                                                                                                     |
| seeds              | see [seeds.txt](https://github.com/atomone-hub/atomone-validator-community/blob/main/atomone-1/seeds.txt)                       |
| persistent_peers   | see [persistent_peers.txt](https://github.com/atomone-hub/atomone-validator-community/blob/main/atomone-1/persistent_peers.txt) |

### Hardware recommendations

AtomOne is a relatively simple and vanilla Cosmos SDK chain with minor modifications. The recommended minimum hardware requirements should be enough to comfortably be able to run a validator node.

| Node Type  | RAM   | Storage*     |
|------------|-------|--------------|
| Validator  | 32GB  | 500GB-2TB    |
| Full       | 16GB  | 2TB          |
| Default    | 16GB  | 1TB          |

> *\* Storage size depend on level of pruning.*

Reviewer: ```suggestion
*\* Storage size depends on level of pruning.*
```
